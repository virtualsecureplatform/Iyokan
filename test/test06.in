[[bits]]
name = "wren"
size = 16
bytes = [0b10110010, 0b01000101]

[[bits]]
name = "addr"
size = 128   # 8 * 16
bytes = [0, 128, 31, 237, 92, 124, 91, 188, 176, 105, 47, 225, 97, 170, 144, 128]

[[bits]]
name = "wdata"
size = 128   # 8 * 16
bytes = [0, 29, 245, 2, 110, 124, 166, 49, 230, 51, 163, 16, 165, 218, 59, 255]

[[ram]]
name = "target"
size = 2048 # 256 * 8
bytes = []
