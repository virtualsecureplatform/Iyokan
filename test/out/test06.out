rom = []
cycles = 16
bits = [
{bytes=[29],size=8,name="rdata"},
]
[[ram]]
bytes = [
0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,0,0,163,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,110,0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,124,0,0,0,29,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,59,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
0,0,0,0,230,0,0,0,0,0,0,0,0,0,0,0,49,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
]
size = 2048
name = "target"

